tests/graph-container.js

```diff
@@ -74,16 +74,16 @@ describe('@graph container', () => {
       jsonld.compact(expanded, doc['@context'], (err, compacted) => {
         assert.ifError(err);
         assert.deepEqual(compacted, {
-          "@context": {
-            "@version": 1.1,
-            "input": {
-              "@id": "foo:input",
-              "@container": "@graph"
+          '@context': {
```

Comment:
Switching to single quotes means this is no longer copy/paste friendly. 😢 Could you switch object keys back to double quotes?

Reply:
Yeah, I always debate the wisdom of making otherwise pure json data use any format but json.  I can change this back.